SleepEngine/Engine/Utils/file_utils.cpp

```diff
@@ -0,0 +1,22 @@
+#include "stdafx.h"
+#include "file_utils.h"
+
+std::optional<std::string> readFile(std::filesystem::path path)
+{
+    if (!std::filesystem::exists(path))
+    {
+        return std::nullopt;
+    }
+
+    int bytes = static_cast<int>(std::filesystem::file_size(path));
```

Comment:
hey, you definitely should use `auto` here.
file_size returns `std::uintmax_t`.

Reply:
not auto though
i had no idea what this thing returns, and will forget in 2 mins

Reply:
the idea of auto is that you don't even think about it.
But ok